### ISSUE TITLE
Fedora 40 fatal error when login

### ISSUE THREAD

**Issue by sahak1an** [arangodb/arangodb#20909]
## My Environment

* __ArangoDB Version__:        
3.12.0-2 Enterprise
* __Deployment Mode__:       
Single Server
* __Deployment Strategy__:   
 Manual Start
* __Infrastructure__:               
local
* __Operating System__:        
Fedora 40
* __Total RAM in your machine__:        
64GB
* __Disks in use__:        
SSD
* __Used Package__:              
yum
## Component, Query & Data

__Affected feature__:

Login

## Steps to reproduce

1.  yum -y install arangodb3e-3.12.0-1.2
2.  try login

__Problem__:

```
May 08 11:42:56 fedora arangod[6445]: 2024-05-08T07:42:56Z [6445] FATAL [a7902] {crash} 💥 ArangoDB 3.12.0-2 enterprise [linux], build-id cea400270e1534a52971f725616f03e439b9fe93, thread 68 [SchedWorker] caught unexpected signal 8 (SIGFPE, sub type FPE_INTDIV: integer divide by zero) from pid 18446744072696074417 in state "in wait": signal handler invoked - image base address: 0x0000000000200000 - CPU context: rip: 0x00007f1ec39790b1, rsp: 0x00007f1e915fa750, efl: 0x0000000000010297, rbp: 0x00007f1e915fa780, rsi: 0x0000000000001000, rdi: 0x0000000000800000, rax: 0xffffffffffffffff, rbx: 0x0000000000000000, rcx: 0xffffffffffffffff, rdx: 0x0000000000000000, r8: 0x0000000000000000, r9: 0x00007f1e39ec7a00, r10: 0x00007f1e915fa750, r11: 0x0000000000000246, r12: 0x0000000000000018, r13: 0x0000000000000001, r14: 0x00007f1e39e28800, r15: 0x00007f1e39e29200
May 08 11:43:00 fedora systemd[1]: arangodb3.service: Main process exited, code=dumped, status=8/FPE
May 08 11:43:00 fedora systemd[1]: arangodb3.service: Failed with result 'core-dump'.

```

**Comment by dothebart** [comment]
Hi, 
can you have a look at `/var/log/arangodb3/arangod.log` ? it should list a stacktrace of this crash.

**Comment by sahak1an** [comment]
```
2024-05-08T07:35:05Z [5694] INFO [6ea38] {general} using endpoint 'http+tcp://127.0.0.1:8529' for non-encrypted requests
2024-05-08T07:35:05Z [5694] INFO [cf3f4] {general} ArangoDB (version 3.12.0-2 enterprise [linux]) is ready for business. Have fun!
2024-05-08T07:42:38Z [5694] INFO [b4133] {general} SIGTERM received (sender pid 1), beginning shut down sequence
2024-05-08T07:42:38Z [5694] INFO [4bcb9] {general} ArangoDB has been shut down
2024-05-08T07:42:39Z [6445] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2024-05-08T07:42:39Z [6445] INFO [25362] {memory} Available physical memory: 67222175744 bytes, available cores: 20
2024-05-08T07:42:39Z [6445] INFO [3bb7d] {cluster} Starting up with role SINGLE
2024-05-08T07:42:39Z [6445] INFO [f6e0e] {aql} memory limit per AQL query automatically set to 40333305447 bytes. to modify this value, please adjust the startup option `--query.memory-limit`
2024-05-08T07:42:39Z [6445] INFO [fe333] {engines} RocksDB recovery starting, scanning WAL starting from sequence number 2499, latest sequence number: 2574, active log files: 17, files in archive: 0
2024-05-08T07:42:39Z [6445] INFO [a4ec8] {engines} RocksDB recovery finished, WAL entries scanned: 76, recovery start sequence number: 2499, latest WAL sequence number: 2574, max tick value found in WAL: 0, last HLC value found in WAL: 0
2024-05-08T07:42:39Z [6445] INFO [c1b63] {arangosearch} ArangoSearch maintenance: [3..3] commit thread(s), [3..3] consolidation thread(s). ArangoSearch execution parallel threads limit: 40
2024-05-08T07:42:39Z [6445] INFO [c2c74] {arangosearch} ArangoSearch columns cache limit: 0
2024-05-08T07:42:56Z [6445] FATAL [a7902] {crash} 💥 ArangoDB 3.12.0-2 enterprise [linux], build-id cea400270e1534a52971f725616f03e439b9fe93, thread 68 [SchedWorker] caught unexpected signal 8 (SIGFPE, sub type FPE_INTDIV: integer divide by zero) from pid 18446744072696074417 in state "in wait": signal handler invoked - image base address: 0x0000000000200000 - CPU context: rip: 0x00007f1ec39790b1, rsp: 0x00007f1e915fa750, efl: 0x0000000000010297, rbp: 0x00007f1e915fa780, rsi: 0x0000000000001000, rdi: 0x0000000000800000, rax: 0xffffffffffffffff, rbx: 0x0000000000000000, rcx: 0xffffffffffffffff, rdx: 0x0000000000000000, r8: 0x0000000000000000, r9: 0x00007f1e39ec7a00, r10: 0x00007f1e915fa750, r11: 0x0000000000000246, r12: 0x0000000000000018, r13: 0x0000000000000001, r14: 0x00007f1e39e28800, r15: 0x00007f1e39e29200
2024-05-08T07:43:05Z [6865] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2024-05-08T07:43:05Z [6865] INFO [25362] {memory} Available physical memory: 67222175744 bytes, available cores: 20
2024-05-08T07:43:05Z [6865] INFO [3bb7d] {cluster} Starting up with role SINGLE
2024-05-08T07:43:05Z [6865] INFO [f6e0e] {aql} memory limit per AQL query automatically set to 40333305447 bytes. to modify this value, please adjust the startup option `--query.memory-limit`
2024-05-08T07:43:05Z [6865] WARNING [b387d] {general} found existing lockfile '/var/lib/arangodb3/LOCK' of previous process with pid 6445, but that process seems to be dead already
2024-05-08T07:43:05Z [6865] INFO [fe333] {engines} RocksDB recovery starting, scanning WAL starting from sequence number 2614, latest sequence number: 2652, active log files: 18, files in archive: 0
2024-05-08T07:43:05Z [6865] INFO [a4ec8] {engines} RocksDB recovery finished, WAL entries scanned: 39, recovery start sequence number: 2614, latest WAL sequence number: 2652, max tick value found in WAL: 0, last HLC value found in WAL: 0
2024-05-08T07:43:06Z [6865] INFO [c1b63] {arangosearch} ArangoSearch maintenance: [3..3] commit thread(s), [3..3] consolidation thread(s). ArangoSearch execution parallel threads limit: 40
2024-05-08T07:43:06Z [6865] INFO [c2c74] {arangosearch} ArangoSearch columns cache limit: 0
2024-05-08T07:56:56Z [6865] FATAL [a7902] {crash} 💥 ArangoDB 3.12.0-2 enterprise [linux], build-id cea400270e1534a52971f725616f03e439b9fe93, thread 68 [SchedWorker] caught unexpected signal 8 (SIGFPE, sub type FPE_INTDIV: integer divide by zero) from pid 18446744072024985777 in state "in wait": signal handler invoked - image base address: 0x0000000000200000 - CPU context: rip: 0x00007fc89b9790b1, rsp: 0x00007fc869ffa750, efl: 0x0000000000010297, rbp: 0x00007fc869ffa780, rsi: 0x0000000000001000, rdi: 0x0000000000800000, rax: 0xffffffffffffffff, rbx: 0x0000000000000000, rcx: 0xffffffffffffffff, rdx: 0x0000000000000000, r8: 0x0000000000000000, r9: 0x00007fc7cc5c1e00, r10: 0x00007fc869ffa750, r11: 0x0000000000000246, r12: 0x0000000000000018, r13: 0x0000000000000001, r14: 0x00007fc810da5300, r15: 0x00007fc810da5d00
2024-05-08T07:57:08Z [11172] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2024-05-08T07:57:08Z [11172] INFO [25362] {memory} Available physical memory: 67222175744 bytes, available cores: 20
2024-05-08T07:57:08Z [11172] INFO [3bb7d] {cluster} Starting up with role SINGLE
2024-05-08T07:57:08Z [11172] INFO [f6e0e] {aql} memory limit per AQL query automatically set to 40333305447 bytes. to modify this value, please adjust the startup option `--query.memory-limit`
2024-05-08T07:57:08Z [11172] WARNING [b387d] {general} found existing lockfile '/var/lib/arangodb3/LOCK' of previous process with pid 6865, but that process seems to be dead already
2024-05-08T07:57:08Z [11172] INFO [fe333] {engines} RocksDB recovery starting, scanning WAL starting from sequence number 6313798, latest sequence number: 6313847, active log files: 6, files in archive: 0
2024-05-08T07:57:08Z [11172] INFO [a4ec8] {engines} RocksDB recovery finished, WAL entries scanned: 50, recovery start sequence number: 6313798, latest WAL sequence number: 6313847, max tick value found in WAL: 0, last HLC value found in WAL: 0
2024-05-08T07:57:08Z [11172] INFO [c1b63] {arangosearch} ArangoSearch maintenance: [3..3] commit thread(s), [3..3] consolidation thread(s). ArangoSearch execution parallel threads limit: 40
2024-05-08T07:57:08Z [11172] INFO [c2c74] {arangosearch} ArangoSearch columns cache limit: 0
2024-05-08T08:45:57Z [11172] INFO [55df5] {replication} no collection count adjustment needs to be applied for _system/grouping: counted value: 0
2024-05-08T08:45:57Z [11172] INFO [55df5] {replication} no collection count adjustment needs to be applied for _system/grouping: counted value: 0
2024-05-08T08:45:57Z [11172] INFO [55df5] {replication} no collection count adjustment needs to be applied for _system/grouping: counted value: 0
2024-05-08T11:04:51Z [11172] FATAL [a7902] {crash} 💥 ArangoDB 3.12.0-2 enterprise [linux], build-id cea400270e1534a52971f725616f03e439b9fe93, thread 70 [SchedWorker] caught unexpected signal 8 (SIGFPE, sub type FPE_INTDIV: integer divide by zero) from pid 22515889 in state "in wait": signal handler invoked - image base address: 0x0000000000200000 - CPU context: rip: 0x00007f4a015790b1, rsp: 0x00007f49cddfa750, efl: 0x0000000000010297, rbp: 0x00007f49cddfa780, rsi: 0x0000000000001000, rdi: 0x0000000000800000, rax: 0xffffffffffffffff, rbx: 0x0000000000000000, rcx: 0xffffffffffffffff, rdx: 0x0000000000000000, r8: 0x0000000000000000, r9: 0x00007f494f235800, r10: 0x00007f49cddfa750, r11: 0x0000000000000246, r12: 0x0000000000000018, r13: 0x0000000000000001, r14: 0x00007f494f2b9a00, r15: 0x00007f494f2ba400
2024-05-08T11:05:00Z [21341] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2024-05-08T11:05:00Z [21341] INFO [25362] {memory} Available physical memory: 67222175744 bytes, available cores: 20
2024-05-08T11:05:00Z [21341] INFO [3bb7d] {cluster} Starting up with role SINGLE
2024-05-08T11:05:00Z [21341] INFO [f6e0e] {aql} memory limit per AQL query automatically set to 40333305447 bytes. to modify this value, please adjust the startup option `--query.memory-limit`
2024-05-08T11:05:00Z [21341] WARNING [b387d] {general} found existing lockfile '/var/lib/arangodb3/LOCK' of previous process with pid 11172, but that process seems to be dead already
2024-05-08T11:05:00Z [21341] INFO [fe333] {engines} RocksDB recovery starting, scanning WAL starting from sequence number 6542691, latest sequence number: 6542694, active log files: 7, files in archive: 0
2024-05-08T11:05:00Z [21341] INFO [a4ec8] {engines} RocksDB recovery finished, WAL entries scanned: 4, recovery start sequence number: 6542691, latest WAL sequence number: 6542694, max tick value found in WAL: 0, last HLC value found in WAL: 0
2024-05-08T11:05:01Z [21341] INFO [c1b63] {arangosearch} ArangoSearch maintenance: [3..3] commit thread(s), [3..3] consolidation thread(s). ArangoSearch execution parallel threads limit: 40
2024-05-08T11:05:01Z [21341] INFO [c2c74] {arangosearch} ArangoSearch columns cache limit: 0
2024-05-08T11:05:47Z [21341] FATAL [a7902] {crash} 💥 ArangoDB 3.12.0-2 enterprise [linux], build-id cea400270e1534a52971f725616f03e439b9fe93, thread 68 [SchedWorker] caught unexpected signal 8 (SIGFPE, sub type FPE_INTDIV: integer divide by zero) from pid 18446744072891109553 in state "in wait": signal handler invoked - image base address: 0x0000000000200000 - CPU context: rip: 0x00007fddcf3790b1, rsp: 0x00007fdd9cffa750, efl: 0x0000000000010297, rbp: 0x00007fdd9cffa780, rsi: 0x0000000000001000, rdi: 0x0000000000800000, rax: 0xffffffffffffffff, rbx: 0x0000000000000000, rcx: 0xffffffffffffffff, rdx: 0x0000000000000000, r8: 0x0000000000000000, r9: 0x00007fdd412d0a00, r10: 0x00007fdd9cffa750, r11: 0x0000000000000246, r12: 0x0000000000000018, r13: 0x0000000000000001, r14: 0x00007fdd41227900, r15: 0x00007fdd41228300
2024-05-08T11:05:56Z [21754] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2024-05-08T11:05:56Z [21754] INFO [25362] {memory} Available physical memory: 67222175744 bytes, available cores: 20
2024-05-08T11:05:56Z [21754] INFO [3bb7d] {cluster} Starting up with role SINGLE
2024-05-08T11:05:56Z [21754] INFO [f6e0e] {aql} memory limit per AQL query automatically set to 40333305447 bytes. to modify this value, please adjust the startup option `--query.memory-limit`
2024-05-08T11:05:56Z [21754] WARNING [b387d] {general} found existing lockfile '/var/lib/arangodb3/LOCK' of previous process with pid 21341, but that process seems to be dead already
2024-05-08T11:05:56Z [21754] INFO [fe333] {engines} RocksDB recovery starting, scanning WAL starting from sequence number 6542856, latest sequence number: 6542910, active log files: 8, files in archive: 0
2024-05-08T11:05:56Z [21754] INFO [a4ec8] {engines} RocksDB recovery finished, WAL entries scanned: 55, recovery start sequence number: 6542856, latest WAL sequence number: 6542910, max tick value found in WAL: 0, last HLC value found in WAL: 0
2024-05-08T11:05:57Z [21754] INFO [c1b63] {arangosearch} ArangoSearch maintenance: [3..3] commit thread(s), [3..3] consolidation thread(s). ArangoSearch execution parallel threads limit: 40
2024-05-08T11:05:57Z [21754] INFO [c2c74] {arangosearch} ArangoSearch columns cache limit: 0
2024-05-08T11:05:59Z [21754] FATAL [a7902] {crash} 💥 ArangoDB 3.12.0-2 enterprise [linux], build-id cea400270e1534a52971f725616f03e439b9fe93, thread 68 [SchedWorker] caught unexpected signal 8 (SIGFPE, sub type FPE_INTDIV: integer divide by zero) from pid 18446744073509769393 in state "in wait": signal handler invoked - image base address: 0x0000000000200000 - CPU context: rip: 0x00007f58f41790b1, rsp: 0x00007f58c23fa750, efl: 0x0000000000010297, rbp: 0x00007f58c23fa780, rsi: 0x0000000000001000, rdi: 0x0000000000800000, rax: 0xffffffffffffffff, rbx: 0x0000000000000000, rcx: 0xffffffffffffffff, rdx: 0x0000000000000000, r8: 0x0000000000000000, r9: 0x00007f5866ad7a00, r10: 0x00007f58c23fa750, r11: 0x0000000000000246, r12: 0x0000000000000018, r13: 0x0000000000000001, r14: 0x00007f5866a27e00, r15: 0x00007f5866a28800
2024-05-08T11:06:08Z [22152] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2024-05-08T11:06:08Z [22152] INFO [25362] {memory} Available physical memory: 67222175744 bytes, available cores: 20
2024-05-08T11:06:08Z [22152] INFO [3bb7d] {cluster} Starting up with role SINGLE
2024-05-08T11:06:08Z [22152] INFO [f6e0e] {aql} memory limit per AQL query automatically set to 40333305447 bytes. to modify this value, please adjust the startup option `--query.memory-limit`
2024-05-08T11:06:08Z [22152] WARNING [b387d] {general} found existing lockfile '/var/lib/arangodb3/LOCK' of previous process with pid 21754, but that process seems to be dead already
2024-05-08T11:06:08Z [22152] INFO [fe333] {engines} RocksDB recovery starting, scanning WAL starting from sequence number 6542913, latest sequence number: 6542967, active log files: 9, files in archive: 0
2024-05-08T11:06:08Z [22152] INFO [a4ec8] {engines} RocksDB recovery finished, WAL entries scanned: 59, recovery start sequence number: 6542913, latest WAL sequence number: 6542967, max tick value found in WAL: 29, last HLC value found in WAL: 1798482278018449408
2024-05-08T11:06:08Z [22152] INFO [c1b63] {arangosearch} ArangoSearch maintenance: [3..3] commit thread(s), [3..3] consolidation thread(s). ArangoSearch execution parallel threads limit: 40
2024-05-08T11:06:08Z [22152] INFO [c2c74] {arangosearch} ArangoSearch columns cache limit: 0
2024-05-08T12:23:57Z [22152] FATAL [a7902] {crash} 💥 ArangoDB 3.12.0-2 enterprise [linux], build-id cea400270e1534a52971f725616f03e439b9fe93, thread 68 [SchedWorker] caught unexpected signal 8 (SIGFPE, sub type FPE_INTDIV: integer divide by zero) from pid 957845681 in state "in wait": signal handler invoked - image base address: 0x0000000000200000 - CPU context: rip: 0x00007f76391790b1, rsp: 0x00007f76073fa750, efl: 0x0000000000010297, rbp: 0x00007f76073fa780, rsi: 0x0000000000001000, rdi: 0x0000000000800000, rax: 0xffffffffffffffff, rbx: 0x0000000000000000, rcx: 0xffffffffffffffff, rdx: 0x0000000000000000, r8: 0x0000000000000000, r9: 0x00007f75aca97000, r10: 0x00007f76073fa750, r11: 0x0000000000000246, r12: 0x0000000000000018, r13: 0x0000000000000001, r14: 0x00007f75acabf400, r15: 0x00007f75acabfe00
2024-05-08T12:24:06Z [24529] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2024-05-08T12:24:06Z [24529] INFO [25362] {memory} Available physical memory: 67222175744 bytes, available cores: 20
2024-05-08T12:24:06Z [24529] INFO [3bb7d] {cluster} Starting up with role SINGLE
2024-05-08T12:24:06Z [24529] INFO [f6e0e] {aql} memory limit per AQL query automatically set to 40333305447 bytes. to modify this value, please adjust the startup option `--query.memory-limit`
2024-05-08T12:24:06Z [24529] WARNING [b387d] {general} found existing lockfile '/var/lib/arangodb3/LOCK' of previous process with pid 22152, but that process seems to be dead already
2024-05-08T12:24:06Z [24529] INFO [fe333] {engines} RocksDB recovery starting, scanning WAL starting from sequence number 6562686, latest sequence number: 6562740, active log files: 10, files in archive: 0
2024-05-08T12:24:06Z [24529] INFO [a4ec8] {engines} RocksDB recovery finished, WAL entries scanned: 55, recovery start sequence number: 6562686, latest WAL sequence number: 6562740, max tick value found in WAL: 0, last HLC value found in WAL: 0
2024-05-08T12:24:06Z [24529] INFO [c1b63] {arangosearch} ArangoSearch maintenance: [3..3] commit thread(s), [3..3] consolidation thread(s). ArangoSearch execution parallel threads limit: 40
2024-05-08T12:24:06Z [24529] INFO [c2c74] {arangosearch} ArangoSearch columns cache limit: 0
2024-05-08T12:24:12Z [24529] FATAL [a7902] {crash} 💥 ArangoDB 3.12.0-2 enterprise [linux], build-id cea400270e1534a52971f725616f03e439b9fe93, thread 68 [SchedWorker] caught unexpected signal 8 (SIGFPE, sub type FPE_INTDIV: integer divide by zero) from pid 1574408369 in state "in wait": signal handler invoked - image base address: 0x0000000000200000 - CPU context: rip: 0x00007f325dd790b1, rsp: 0x00007f322b9fa750, efl: 0x0000000000010297, rbp: 0x00007f322b9fa780, rsi: 0x0000000000001000, rdi: 0x0000000000800000, rax: 0xffffffffffffffff, rbx: 0x0000000000000000, rcx: 0xffffffffffffffff, rdx: 0x0000000000000000, r8: 0x0000000000000000, r9: 0x00007f31d12d6a00, r10: 0x00007f322b9fa750, r11: 0x0000000000000246, r12: 0x0000000000000018, r13: 0x0000000000000001, r14: 0x00007f31d1227900, r15: 0x00007f31d1228300
2024-05-08T12:24:21Z [24918] INFO [a1c60] {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2024-05-08T12:24:21Z [24918] INFO [25362] {memory} Available physical memory: 67222175744 bytes, available cores: 20
2024-05-08T12:24:21Z [24918] INFO [3bb7d] {cluster} Starting up with role SINGLE
2024-05-08T12:24:21Z [24918] INFO [f6e0e] {aql} memory limit per AQL query automatically set to 40333305447 bytes. to modify this value, please adjust the startup option `--query.memory-limit`
2024-05-08T12:24:21Z [24918] WARNING [b387d] {general} found existing lockfile '/var/lib/arangodb3/LOCK' of previous process with pid 24529, but that process seems to be dead already
2024-05-08T12:24:21Z [24918] INFO [fe333] {engines} RocksDB recovery starting, scanning WAL starting from sequence number 6562743, latest sequence number: 6562797, active log files: 11, files in archive: 0
2024-05-08T12:24:21Z [24918] INFO [a4ec8] {engines} RocksDB recovery finished, WAL entries scanned: 59, recovery start sequence number: 6562743, latest WAL sequence number: 6562797, max tick value found in WAL: 29, last HLC value found in WAL: 1798487195071283200
2024-05-08T12:24:21Z [24918] INFO [c1b63] {arangosearch} ArangoSearch maintenance: [3..3] commit thread(s), [3..3] consolidation thread(s). ArangoSearch execution parallel threads limit: 40
2024-05-08T12:24:21Z [24918] INFO [c2c74] {arangosearch} ArangoSearch columns cache limit: 0
```

**Comment by dothebart** [comment]
Ups, Floating point doesn't have this signal handler :/ 
can you please have a look at the coredump using gdb? 
```
gdb -c [coredumpfilename] /usr/sbin/arangod
bt
```
and post the result? tnx.
